Restaurant.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import RestaurantModel from './RestaurantModel'
+
+class Restaurant extends React.Component {
+  constructor(props) {
+    super(props)
+    this.displayName = "Restaurant"
+
+    this.model = props.model
```

Comment:
I haven't seen this pattern before, and I'm afraid it could cause bugs in some edge cases